opm/autodiff/SimulatorBase_impl.hpp

```diff
@@ -439,6 +441,7 @@ namespace Opm
 
                             const int ok_resv =
                                 well_controls_add_new(RESERVOIR_RATE, target,
+                                                      -std::numeric_limits<int>::max(), -std::numeric_limits<int>::max(),
```

Comment:
Use constants for invalid alq etc. Also int is used for both arguments.

Also line 452.